docs/_data/table/master-configuration.csv

```diff
@@ -4,7 +4,7 @@ alluxio.master.audit.logging.queue.capacity,"10000"
 alluxio.master.backup.abandon.timeout,"1min"
 alluxio.master.backup.connect.interval.max,"30sec"
 alluxio.master.backup.connect.interval.min,"1sec"
-alluxio.master.backup.delegation.enabled,"false"
```

Comment:
@jenoudet tagging you wrt release notes for

alluxio.master.backup.delegation.enabled
alluxio.master.periodic.block.integrity.check.repair
alluxio.master.startup.block.integrity.check.enabled

Reply:
alluxio.master.backup.delegation.enabled : `bin/alluxio fsadmin backup` delegates the backup to a standby master if one exists by default. 
alluxio.master.startup.block.integrity.check.enabled: block integrity and repair does not happen automatically anymore on startup and failover
alluxio.master.periodic.block.integrity.check.repair: master will periodically repair its block knowledge